packages/strapi-hook-mongoose/lib/index.js

```diff
@@ -87,17 +89,33 @@ module.exports = function(strapi) {
         connectOptions.dbName = database;
         connectOptions.useCreateIndex = true;
         connectOptions.useUnifiedTopology = useUnifiedTopology || 'false';
+        connectOptions.reconnectInterval = reconnectInterval || defaults.reconnectInterval;
```

Comment:
the defaults are already set on line 65 (coming from strapi.config.hook.settings.mongoose)